src/Qbead.h

```diff
@@ -74,14 +74,17 @@ float sign(float x) {
   else return -1;
 }
 
+int roundToInt(float x) {
+  if (x > 0) return (int)(x + 0.5);
```

Comment:
could you add the "to avoid precision issues" comment here, otherwise it is a bit confusing that there is a +0.5 here?

Reply:
Yes, I added it